src/plugins/inputs/whois.js

```diff
@@ -0,0 +1,10 @@
+"use strict";
+
+exports.commands = ["whois"];
+
+exports.input = function(network, chan, cmd, args) {
+	// This queries server of the other user and not of the current user, which
+	// does not know idle time.
+	// See http://superuser.com/a/272069/208074.
+	network.irc.raw("WHOIS", args[0], args[0]);
```

Comment:
`args.join(" ")` seems like it'd combine all args into the trailing, so would it end up sending out something like this?
```
WHOIS :arg1 arg2
```

That wouldn't go well. If that's how it'd send out with the above, you'd need to make it push all args separately instead (not sure how you'd do that in JS, but in other langs it'd look something like `network.irc.raw("WHOIS", args...);`